tests/security/cc/cc_audit_test_setup.pm

```diff
@@ -23,6 +23,10 @@ sub run {
 
     select_console 'root-console';
 
+    if (script_run('which SUSEConnect') != 0) {
```

Comment:
I would suggest use rpm -ql SUSEConnect rather than which here, but not a big deal

Reply:
I prefer 'which', seems like easier.